CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 6.11.1 - 2022/04/21
+
+## Enhancements
```

Comment:
nit: If we're calling this 6.11.1 (which I agree with) then I would change this to Fixes